### PR TITLE
Add pre and post_tasks

### PR DESCRIPTION
**Pull request by dabcoder**
Attempts to address #202.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -19,3 +19,9 @@
 
 - include_tasks: agent6-win.yml
   when: not datadog_agent5 and ansible_os_family == "Windows"
+
+- include_tasks: post_tasks/*.yml
```
Comment (hush-hush) [inline tasks/main.yml:23]: Should we ignore errors in post/pre tasks ?

We should also document this in the README

**Review by hush-hush — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (dabcoder) [reply tasks/main.yml:23]: Good point. I would say we shoudn't, since it wouldn't give users any hints as to where the issue(s) lie should they come across any while adding pre and post tasks (and running their playbooks)?

File: README.md [hunk 2]
```diff
@@ -268,6 +268,11 @@ datadog_config_ex:
     custom_sensitive_words: "consul_token,dd_api_key"
 ```
 
+Additional tasks
+----------------
+
+`pre_tasks` and `post_tasks` folders allow to run user defined tasks. `pre_tasks` for tasks to be executed before executing any tasks defined in the `tasks` folder. `post_tasks` for those to be executed after the ones defined in the `tasks` folder.
```
Comment (hush-hush) [inline README.md]: `any tasks defined in the `tasks` folder` I don't think users are aware of the layout of our role.
I would rephrase to something like:

`pre_tasks` and `post_tasks` folders allow to run user defined tasks. `pre_tasks` for tasks to be executed before executing any tasks from the Datadog role and `post_tasks` for those to be executed after.

**Review by hush-hush — APPROVED** [review]
